setup.py

```diff
@@ -1,26 +1,58 @@
+import os
+import platform
 import setuptools
-from lightwood.version import lightwood_version
+
+
+def remove_requirement(requirements, name):
+    return [x for x in requirements if name != x.split(' ')[0]]
+
+os = platform.system()
+
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
 with open('requirements.txt') as req_file:
     requirements = req_file.read().splitlines()
-print(lightwood_version)
+
+dependency_links = []
+
+# Linux specific requirements
+if os == 'Linux':
+    requirements = remove_requirement(requirements,'torch')
+    requirements.append('torch == 1.1.0')
+
+# OSX specific requirements
+if os == 'Darwin':
+    requirements = requirements
+
+# Windows specific requirements
+if os == 'Windows':
+    requirements = remove_requirement(requirements,'torch')
+    requirements = remove_requirement(requirements,'torchvision')
+    requirements.append('torch == 1.1.0.0')
+    requirements.append('torchvision == 0.3.0.0')
+
+    #dependency_links.append('https://download.pytorch.org/whl/cpu/torch-1.1.0-cp37-cp37m-win_amd64.whl#egg=torch-1.1.0.0')
+    #dependency_links.append('https://download.pytorch.org/whl/cpu/torchvision-0.3.0-cp37-cp37m-win_amd64.whl#egg=torchvision-0.3.0.0')
+    dependency_links.append('https://download.pytorch.org/whl/cu100/torch-1.1.0-cp37-cp37m-win_amd64.whl#egg=torch-1.1.0.0')
+    dependency_links.append('https://download.pytorch.org/whl/cu100/torchvision-0.3.0-cp37-cp37m-win_amd64.whl#egg=torchvision-0.3.0.0')
+
 setuptools.setup(
     name="lightwood",
-    version=lightwood_version,
+    version='0.7.1',
```

Comment:
@George3d6  why here? if anything move it to an  about.py like in mindsdb,  but we may want to use the version number else where in the code base, its a good pracrtice that the version can be read from a common file.